package/rmt.service

```diff
@@ -5,12 +5,12 @@ Requires=rmt-migration.service
 After=rmt-migration.service
 
 [Service]
-Type=forking
+Type=simple
 User=_rmt
+PrivateTmp=yes
+Environment=RAILS_LOG_TO_STDOUT=1
```

Comment:
Looks good, but I'd suggest making changes to the RPM package as well -- currently it creates an empty `log` directory to store file logs. It can be dropped now.

Reply:
Does https://github.com/SUSE/rmt/pull/129/commits/923488dd56cdbe7ae462969eff3b3cff1c1c2278 look lood?

Reply:
Yep! :+1:

Reply:
Cool. I can't merge until we merge the `parser` gem fix xD